core/factorization/symbolic.hpp

```diff
@@ -44,14 +44,22 @@ namespace factorization {
  * Computes the symbolic Cholesky factorization of the given matrix.
  *
  * @param mtx  the input matrix
+ * @param symmetrize  should we output the pattern of L + L^T or just L?
```

Comment:
nit: please specify which of these two options correspond to `true` and `false`.

Reply:
good point!